transforms/create-element-to-jsx.js

```diff
@@ -0,0 +1,69 @@
+module.exports = function(file, api) {
+  var j = api.jscodeshift;
+  var root = j(file.source);
+
+  root.find(j.CallExpression, {callee: {object: {name: 'React'}, property: {name: 'createElement'}}})
```

Comment:
can you make this fit into an 80 column limit + add trailing commas?

(I think I have to add eslint to this repo :) )